### ISSUE TITLE
[amélioration] coché les case automatiquement

### ISSUE THREAD

**Issue by vcro** [Mokhet/MFF-roster-tracker#55]
Bonjour,

Est ce qu'il est possible de mettre un module qui permet de cocher les case automatiquement lorsqu'on choisit une statistique ?

cordialement.VCRO

**Comment by Mokhet** [comment]
Oui, je pense ajouter une page de setup qui permettrait de choisir certains comportements de l'outil.

-- 
Vilday Laurent
